### ISSUE TITLE
Release as flatpak/in flathub

### ISSUE THREAD

**Issue by commodorable** [Wavesonics/hammer-editor#37]
As of 1.2.0, Hammer only releases as a .deb on linux. While this works for Debian based linux systems, others such as Fedora use RPM instead of deb and cannot easily install it.

Flatpak is a universal packaging system for linux applications, and uploading it to Flathub allows linux users to easily find, download, install, and update the app.

Please and thank you!

[Flatpak website](https://flatpak.org/)

[Flathub website](https://flathub.org/)

[Flathub submission documentation](https://docs.flathub.org/docs/for-app-authors/submission/)

**Comment by Wavesonics** [comment]
Yes! This is definitely in the plans, but will probably be a while.

**Comment by Wavesonics** [comment]
would this be a good alternative? I could release this very easily: https://appimage.org/

**Comment by qerne130** [comment]
Yes, AppImage would be perfect

**Comment by commodorable** [comment]
Agreed. In my opinion, Flatpak would be best, but Appimage is also great because its universal.

**Comment by privacyfreak84** [comment]
I think flatpak would be better as it is better adopted by different DE's app stores on Linux..

**Comment by jlariviere1** [comment]
I'm on Manjaro with my main system, an .Appimage for flatpak would be pretty cool!  Thanks for the great work!

**Comment by Wavesonics** [comment]
I would love to! I still haven't figured out how to packaged it into one. This is an area I'd be quite happy to accept a PR or even just some help figuring it out.

Releasing on other platforms isn't my top priority until we get closer to a 1.0 release, but if it can be some some what easily I'd be all for it.

**Comment by SpiderUnderUrBed** [comment]
Hello, sorry to ping everyone, but I am curious if this is still planned?

**Comment by Wavesonics** [comment]
I have a flatpak release now that you can download from github or [the site](https://hammer.ink/#download). 

I explored flathub and unfortunately our build process isn't compatible with their requirements.
